src/pages/HowItWorks/HowItWorks.jsx

```diff
@@ -19,6 +20,7 @@ import "./how-it-works.scss";
  * @returns {JSX.Element}
  */
 export const HowItWorks = () => {
+  const { t } = useTranslation("how-it-works");
```

Comment:
@georgipavlov-7DIGIT How does that work if you haven't added a locale for the page, you haven't exported it, and you haven't imported it in the i18n file?!